### DISCUSSION TITLE
Symbol便利ツールやゲームの情報求む

### DISCUSSION THREAD

**Discussion by ymuichiro** [ymuichiro/symbol_japan_forum#21] [Ideas]
以下の情報をご提供いただけないでしょうか
Symbol/NEM関連のゲーム or 便利ツールについての

- URL
- 名前
- 作者
- 説明文

頂いた情報を元にWikiに記載し誰でも参照できるようにまとめていきたく...

**Comment by 0x070696E65** [comment]
- https://symvolution.com/thetower/
- The Tower
- toshi 
- キャラクター育成型の2D見下ろしアクションゲーム！ パラメータはライフ・パワー・インテリジェンスの３種類。育成方針によって戦闘スタイルも変わる！ランダム生成されるダンジョンでコインを集め、キャラを育成し、他プレイヤーと競いながら無限タワーの記録に挑戦 。階が上がれば上がるほど敵は強くなるので育成しながら頂上を目指せ。街に帰れば育成だけでなく、様々なNPCプレイヤーがお出迎え。委任ハーベスターがお得なギャンブルや現実世界のSymbolモザイクを交換できる交換所。NFT-DriveとCOMSAに対応した3D美術館など Symbolerの暇を全力で潰すことを目的としたゲームです。

**Comment by 0x070696E65** [comment]
- https://symvolution.com/account/owner
- 俺の勇者
- toshi
- 上記TheTowerのマイ勇者お披露目ツール。SSS_Extensionとの連携で自分の勇者のパラメータや攻略階層、討伐モンスター、所有装飾品などを表示できる。さらに装飾品の販売が可能。

**Comment by 0x070696E65** [comment]
- https://symbol-tank.netlify.app/
- Symbol Tank
- planet★箒星
- 未承認トランザクションによって火力がパワーアップする2D(戦)車ゲームです。トランザクションの種類によってパワーアップの種類が決まります。ゲーム配信において視聴者がリアルタイムでヨイショすることができるかもしれません。ステージは25まであり、その後はランダムで登場し、エンディングはありません。また、音はありませんので周囲を気にせずにプレイできます。

**Comment by 0x070696E65** [comment]
- https://owlfeathernemtushackathon.ml/
- The edge of XYMverse
- OwlFeather
- 出来かけのワールドシミュレーションゲーム？（ノベルゲーム？）です。バグありかつ未完成で出品するのを心苦しく思う気持ちもありますが、一方でクソゲーとして楽しめなくもないかなと開き直ってもいます。

**Comment by 0x070696E65** [comment]
- https://nemtus.nftowner.shop/
- めちゃラクなNemFTシャツ屋さん
- めちゃラク
- NFTアート買ったけど、どうすればいい？問題を解決する画期的サービス。自分の権利を持つNFTアートを印刷したTシャツを手間なく作れて、その売上で競うNFTマネーゲーム。新Comsaと連携！ 初めてのSymbolでしたが試行錯誤を繰り返しなんとか出来ました！

**Comment by 0x070696E65** [comment]
- https://tetsym-2c688.web.app/
- NFTgame～TETSYM～
- Nemoob
- NFTゲームにするつもりのものです！！ 見かけ上わかりませんがブロックチェーン上にあるコードを ブラウザに読み込ませることで、 プレイできる「テトリス」となっております！！ 読み込む都合上画面が反映されないことがあります。 その際はページ更新してください。 手法としてはSymbolのテストネット上に複数のアカウントを用意し、 そのアカウントにコードを分割してトランザクションを送信することで 擬似的にhtmlファイルをブロックチェーン上に載せています。 その後モザイクのメタデータをもとにしてアカウントからコードを読み出し、 ブラウザに反映させます。 遊んでもらう都合上アドレスはべた書きです！

**Comment by 0x070696E65** [comment]
- http://ueno.tokyo/mitsukagi/index.html
- この密室は鍵が閉まっている -MISSHITU DAMONO-
- なべち農園
- 密室からの脱出。鍵の閉まった密室。封鎖された空間。 信じられるのは自分の感覚と記憶のみ。 この密室から、あなたは脱出できるのか。

**Comment by 0x070696E65** [comment]
- https://teritaris.github.io/tenxym/
- 異世界転生したら法定通貨がXYMだった件
- てりあ
- "全プレイヤー協力型オンラインRPG！一部敵のHPはモザイク残高で管理されているためゲーム外のSymbolExplorer等で敵のHPが見える！ アバター変更、チャット、オンチェーン掲示板、保有しているモザイクによって変化するイベント、クリア特典でモザイク自動配信などを取り入れたメインネット対応ゲームです。"

**Comment by 0x070696E65** [comment]
- https://kurikou02.github.io/symboltyping/
- Symbol Typing Quest
- マロン
- Symbolの世界観やキーワードに触れながら楽しくタイピングの練習ができるゲームです。

**Comment by 0x070696E65** [comment]
- https://taroventure.com/
- たろうくんの大冒険
- Teamたろう
- 宝探しイベント前日。ハリネズミのたろうくんが朝目を覚ますと、準備していた宝探しの道具が姿を消していた！友達のさちこと一緒に道具を探す冒険の旅へ出発だ！（プレイ時間目安：5 ～ 10分）。

**Comment by 0x070696E65** [comment]
- http://symbolxym.html.xdomain.jp/
- ブロック崩し
- NoーIchi
- Excel Wordすら触ったことのない初心者が作ったブロック崩しです！　オリジナルのゲームを作っていましたが期限に間に合わなかったので、プログラミングチャレンジのブロック崩しを自分なりにアレンジして作りました！

**Comment by 0x070696E65** [comment]
- https://gamebook-in-symbol.vercel.app/
- The Gamebook System in Symbol
- scrp
- 本作品はSymbolブロックチェーンを使って作成されたGamebookシステムです。 ブロックチェーン上へ番地の登録、メッセージの書き込みをすることで、誰でも、自由に本文や選択肢、エンディングを書き込むことができます。 使い方・遊び方はこちら：https://bit.ly/3HNVZIQ 本システムの初期アイディアは梧桐重吾様(@gotofifteen)からいただき、提出時点で登録さているゲームブック本文も協力いただきました。 本作品が、ゲームブックや、ブロックチェーンに触れるきっかけになると幸いです。

**Comment by 0x070696E65** [comment]
- http://xymtre.symbol-farm.tokyo/
- 脳トレXYM
- SHIN
- 当施設は時間をSymbolメインチェーンのブロック生成タイミングに支配された脳 力トレーニングXYMです。当施設はトレーニングのための施設ですので、機械に 頼らず、何かをいじったりせず、自身の脳力のみでご利用頂きますようお願いします。

**Comment by 0x070696E65** [comment]
- https://uneatingmeals.com/
- Uneatingmeals
- KenshiHashiguchi
- Mealsと呼ばれるNFTをゲットしてゲームで遊んでみよう！ このNFTの特徴はオンチェーンNFTとなっており、 #Garush の設計をsvg形式に拡張した作りになってます。 今後はもっと本格的なゲームを作成し、ゲームランキング上位者に色やパーツの変更券や、Mealsのレベルアップ券を配布し、ゲームに参加することでMealsが育っていくようなシステムを構想しています。（DAOの構築も検討中）育てたMealsはNFTなので、当然売ることもできます。Play to earnの第一歩です！

**Comment by 0x070696E65** [comment]
- https://github.com/naka-sho/hobby
- 簡易仮想通貨送信システム
- s-nakao
- ユーザのアプリにAPIをリクエストする実装を作るだけで、簡単にsymbolを送金できます。 主にツイッターなどでよくあるsymbolのばらまき企画の際に、ゲーム感覚で使用できます。 システムがわかる人は、conohaなどレンタルサーバーを借りればチャット画面を公開することもできます。 特定のアプリに組み込むシステムのため、作品リンクはgithubにしています。

**Comment by 0x070696E65** [comment]
- https://mirai-nft-fishing.tk/
- 夢　未来　NFT FISHING
- ボー太郎
- 未来ちゃんのNFT フィッシング！　たくさん釣ってSymbolのNFTをもらっちゃおう！

**Comment by 0x070696E65** [comment]
- https://nabe-3.github.io/NEMTUSHackathon/
- Symball
- nabe3
- "視聴者参加型の実況配信向けゲーム。 特定のアドレスにトランザクションを送るとゲームに影響を与えられます。 視聴者が配信者の手助けしたり助言したり、時には邪魔したり。 トランザクションは以下の通り送ってください。 宛先：NCJELEW7XZAYFS56PXW5RCL5CWABTT5YVLO6BFY XYM送金：0XYM以外 以下のいずれかをメッセージに記述 ・ball_add　ボールを2個追加 ・life_up　自機を2機追加 ・speed_up　移動速度アップ ・speed_down　移動速度ダウン ・score_up　送ったXYMの分だけスコアアップ ・上記以外　画面上にメッセージ表示" 動作環境：Googlechromeシークレットウィンドウ、Firefox、Safari

**Comment by 0x070696E65** [comment]
- https://inatatsu-tatsuhiro.github.io/SymbolDraw/
- Symbol Draw
- inatatsu
- Symbol DrawはSymbolブロックチェーンを用いて、落書きのチェーンを作ることが出来ます。また、Symbol Drawで作成した作品をアポスティーユしたり、モザイクに紐付け、確認することもできます。

**Comment by Cat-008** [comment]
・https://xembook.github.io/xembook/
・XEMBook
・XEMBook

アドレス残高、履歴表示ツール。
ウォレットアドレスを入力するだけで、該当アドレスの出入金、ハーベスト履歴、時価を確認することができます。

**Comment by Cat-008** [comment]
・https://xembook.github.io/xembook/xemtax
・XEMTax
・XEMBook

平均取得単価の調査ツール。ウォレットアドレスを入力するだけで確定申告で必要な取得単価を調べることができる

**Comment by Cat-008** [comment]
・https://t.me/XymHarvestMonitorbot
・HarvestMonitor
・さくたろう

テレグラムアプリにアドレスを登録するとHarvestや着金の通知が来る。複数ウォレットの合計残高を枚数、円で表示可能。

**Reply by nononon00** [reply]
さくたろうさんの利用方法記事おいておきます🙌

https://nemlog.nem.social/blog/59119

**Comment by Cat-008** [comment]
・https://line.me/ti/g2/ay3ZKo6jJomHv4xpm1PFNaceMsR5Mh_pAorMMQ?utm_source=invitation&utm_medium=link_copy&utm_campaign=default
・オープンチャット「Symbol×NEM 初心者歓迎🔰質問・情報共有」
・のののん

Symbolに関するわからないことがあれば、いつでも質問できるLINEのオープンチャット

**Comment by Cat-008** [comment]
【Android】
https://play.google.com/apps/testing/com.shu.software.symbol_arcana
【iPhone】
https://apps.apple.com/jp/app/arcana-wallet/id1603599435
・Arcana
・しゅう

NEMLOGを作ったしゅうさんが作ったモバイルウォレットアプリ。モザイク図鑑、ガチャ、ノード委任、チャット、モザイク発行、色んな機能が利用できるウォレット

**Reply by nononon00** [reply]
オプチャノートに入れておいたのでiOS追記しておきます🙌

◆iOS
https://apps.apple.com/jp/app/arcana-wallet/id1603599435

◆解説動画
https://youtu.be/s9QTbs32iU4
動画のタイムライン
0:30~ : ウォレット登録
1:35~ : ウォレット登録 [プライベートキー]
3:12~ : ウォレット登録 [ニーモニック]
6:50~ : ウォレット登録 [新規]
8:18~ : ウォレット操作
11:43~ : ホーム画面
14:06~ : 送金操作
15:33~ : 着金送金履歴
16:50~ : アドレス登録
19:03~ : ハーベスト・委任
25:05~ : モザイク作成
27:22~ : ノード接続
28:25~ : インボイス発行
30:50~ : チャット
32:01~ : NEMLOGビューワー
33:05~ : NEMber Art
33:36~ : 設定
35:57~ : アグリゲートボンデッド詐欺対策
40:15~ : まとめ

**Reply by Cat-008** [reply]
ありがとうございます💪✨
助かりました😊

**Comment by Cat-008** [comment]
・https://nemlog.nem.social/blog/75284
・MosaicGarden
・だい

SymbolモザイクをMosaicGardenアドレス(NB3YJ6FZ7CRZLMILAW4S6Y3ONUP5TG5GZXGFWNA)に送ると、違うモザイクが受け取れるモザイクの自動交換所

**Comment by nononon00** [comment]
・iOS版
https://t.co/roibujclEg
・Android版
https://t.co/ivm2PX5yah 

・nodewatcher(アプリ)

・こけっこー
https://twitter.com/ccdev/status/1395688713099776005?t=3tkU8Qm2fH3BdXuNxbac9Q&s=19

・ノードに委任している方のアドレスを一覧で確認できます。ハーベスト通知機能もあり。

**Comment by nononon00** [comment]
・https://symbol-tools.com/symbolTools/view/tweetExtract/tweetExtractEntrance.html

・引用リツイートからSYMBOLアドレスを一覧化するやつ。

・ahiki(あひき)
https://twitter.com/ahikicoin/status/1486862221669912579?t=evP6p6ItbDlzgaOX9WOjrg&s=19

・引用リツイートからSYMBOLアドレスを一覧化
配布イベント等で活用できます

**Comment by nononon00** [comment]
・https://aggregate-tx-sender-ulunta.pythonanywhere.com/

・Mosaic sender

・ULUNTA
https://twitter.com/ULUNTA2/status/1489077789261066242?t=aDNKLlcZ6fV1XbLsLFL1cQ&s=19

・指定形式のcsvファイル読み込みでSymbolのモザイクを一括送金するシステム

**Comment by nononon00** [comment]
・https://symbol-tools.com/symbolTools/view/tool/nodeList.html
・Symbol Node List
・ahiki(あひき)
・Symbolのノードと各ノードの空き状況や、Twitter、意気込み(コメント)を一覧化。ブロック高が遅れているノード等の死活状況をスクリーニングしたうえで表示。ノード詳細ページより委任アドレス、残高、委任履歴の確認や委任設定も可能。

**Reply by nononon00** [reply]
アプリも！
https://twitter.com/ahikicoin/status/1478702649243729922?t=ct8VlWj0z7_a8myL1K-0sw&s=19

**Comment by nononon00** [comment]
・https://poi-symbol.com/poi_website/farmers_history
・Farmers History
・poi_symbol
https://mobile.twitter.com/poi_symbol
・アドレス入力で過去一定期間のハーベスト報酬の情報をグラフで可視化

**Comment by nononon00** [comment]
・https://symbol.openapostille.com/
・OpenApostille(オープンアポスティーユ)
・いなたつ
https://twitter.com/_inatatsu_csg_/status/1456865905355354119?t=X1kKLkcFZhJ7TMCeQjBPwA&s=19
・アポスティーユについて
アポスティーユは #NEM で生まれた譲渡(共有)可能なタイムスタンプ公証サービスで、オリジナルのファイルをチェーン上に保存しなくても、ある時点以降のそのファイルの存在や真正性を担保する技術です。
※詳細はホワイトペーパーを参照
https://t.co/tPcqNghCl1
公証とは
公式の証拠。
特定の事実または法律関係の存在をおおやけに証明すること。行政としては不動産の登記、選挙人名簿への登録など。

使用方法は以下を参照
※現在はメインネットで公開されているため、テストネット用のXYMを入手する必要はありません
https://twitter.com/DaokaTrade/status/1476476929054507008?t=5YmxJUT49m0DxvuZWIJtOA&s=19

**Comment by nononon00** [comment]
・https://liff.line.me/1645278921-kWRPP32q/?accountId=711qovvs
・XYMCITY案内所
・kotopapa
https://twitter.com/nemlog1/status/1485468685016137728?t=JBRjbUmLbjmlBXohYJucbA&s=19
・便利ツールを検索できるLINEbot
LINEbot実装済の機能
新着検索
オススメ検索
カテゴリ検索
いいね機能
開発者情報表示

**Comment by nononon00** [comment]
・https://liff.line.me/1645278921-kWRPP32q/?accountId=820mnegn
・XymHarvestChecker
・kotopapa
https://twitter.com/kotopapa8/status/1499529394263445515?t=ufXayiP2IYixxM8clwUU6w&s=19
・ハーベスト委任状態の確認、インポータンスや残高のチェック、ハーベスト時の通知がLINEで受け取れるBot

**Comment by ymuichiro** [comment]
URL https://github.com/symbol/statistics-service
名前 Symbol Statistics Service（統計サービス）

Symbolの公開鍵よりNodeURL等の逆引きが出来るデータベース照会サービス。以下URLでも稼働版が公開されている
[https://symbol.services/](https://symbol.services/nodes/400FB833ABE3551A1C2FF86F6971D7507C90F6A1CC52B0EA55A8DE5EE2E3AF10)

**Comment by Cat-008** [comment]
https://ishidad2.github.io/NEMBook
・NEMBook
・だい
・ウォレットアドレスの入力で履歴や時価を調べることができる。XEMBook for NEMの復刻版
